e2e/e2e_tests.py

```diff
@@ -39,105 +46,119 @@ def _stop_containers():
     docker_compose.clean()
 
 
-class TestE2E(unittest.TestCase):
-    COOKIES = dict()
-    CSRF_TOKEN = ''
-
-    def _login(self, next=None):
-        login_url = self._make_link('/login/')
-        client = requests.session()
-        client.get(login_url)
+class SeleniumTests(unittest.TestCase):
+    # wait maximal 5 minutes
+    wait_time_seconds = 5*60
```

Comment:
While we're at it, let's use `datetime.timedelta(minutes=5).total_seconds()`, here, too. (Unless we want to have compatibility with pre-2.7 python for scripts outside the Docker containers. [`total_seconds` was added in 2.7](https://docs.python.org/2/library/datetime.html#datetime.timedelta).)